pyproject.toml

```diff
@@ -150,7 +150,7 @@ ocient = [
 oracle = ["cx-Oracle>8.0.0, <8.1"]
 pinot = ["pinotdb>=0.3.3, <0.4"]
 playwright = ["playwright>=1.37.0, <2"]
-postgres = ["psycopg2-binary==2.9.6"]
+postgres = ["psycopg2-binary==2.9.9"]
```

Comment:
Personally I'd vote for leaving this open as in `psycopg2-binary>=2.9.6`

Reply:
what about 

`psycopg2-binary>=2.9.6,<2.10` ?

since postgres is the most famous backend for superset itself

and also cause superset do not associate constraint files with release 

so  I don't want to be the guy that make fail future installation in case of breaking change on psycopg2 ( that is very unlikely cause psycopg3 is already out, but we never know )

Reply:
Requirements are ideally intervals rather than specific versions—which _will_ get pinned for CI et al. when you run `pip-compile-multi`. The concern with pinning a version here is it could severely reduce the feasibly space when `pip-compile-multi` tries to freeze all packages.

Reply:
I don't understand your message @john-bodley , what are you talking about ?